components/Input/index.tsx

```diff
@@ -9,21 +9,21 @@ import {
 	InputLabel,
 	OutlinedInput,
 	FormHelperText,
+	OutlinedInputProps,
 } from '@mui/material'
 import { Visibility, VisibilityOff } from '@mui/icons-material'
 import CheckIcon from '@mui/icons-material/Check'
 import ClearIcon from '@mui/icons-material/Clear'
 import { useTheme } from '@mui/material/styles'
 
-export interface InputProps {
+export interface InputProps extends OutlinedInputProps {
```

Comment:
Was there a need to extend the OutlinedInputProps here? Please tell me, cause extending it here caused the validation on this component to malfunction.

Reply:
I was trying to get some properties from the OutlineInputProps like rows and multiline. I will add them as InputProps instead and remove the extended interface